deploy-scripts/deploy.sh

```diff
@@ -7,9 +7,9 @@ if [ "$TRAVIS_BRANCH" == "master" ]; then
     if [[ "$VERSION_NAME" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
I think you need to change this condition because this is always going to be a snapshot now right?

Reply:
Not exactly:
* `^`: start
* `[0-9]+`: one or more numbers
* `\.`: point
* `[0-9]+`: one or more numbers
* `\.`: point
* `[0-9]+`: one or more numbers
* `$`: end

Reply:
That means no -SNAPSHOT allowed, right?

Reply:
Right. And VERSION_NAME can be a release version or a snapshot version: 

```
if [ "$RELEASE_VERSION" != "$LATEST_PUBLISHED_VERSION" ]; then
    sed -i "s/version.*/version=$RELEASE_VERSION/g" gradle.properties
fi
```

I'll improve that part to make it clear :+1: Thanks @AdrianRaFo !!! :raised_hands: